projects/ng-bulma-lib/src/lib/compenents/carousel/carousel-item/bm-carousel-item.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'bm-carousel-item',
+  template: `
+    <div select="[bm-carousel-item]" [style.width.px]="itemContainerWidth" style="padding-right: 15px">
```

Comment:
Lets put `style="padding-right: 15px"` in some css file and not use inline styling.